components/textfield/stories/textfield.stories.js

```diff
@@ -180,10 +215,13 @@ export default {
 		displayLabel: false,
 		labelPosition: "top",
 		labelText: "Username",
+		displayHelpText: false,
+		helpText: "Create a username with at least 5 characters",
+		placeholder: "Username",
 		size: "m",
 		multiline: false,
 		grows: false,
-		isQuiet: false,
+		pattern: "",
```

Comment:
What would `pattern` be for? I see it as an attribute, but is that for, like, client-side validation or something? I'll have to look it up!

Reply:
Yes, I don't think I've used it before but it looks like you can use `pattern` to trigger `:valid` or `:invalid` pseudoclass which potentially could be neat.